contacktbook/LoggerClass.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace contacktbook
+{
+    public class LoggerClass
+    {
+        public void writeLog(string exceptionText)
+        {
+            using (StreamWriter writer = File.AppendText("Log.txt"))
+            {
+                writer.WriteLine((DateTime.Now.ToString()) + exceptionText);
+            }	
+        }
+    }

```

Comment:
I would organize it a bit different. 
public static void WriteLog (string exceptionText, string exceptionMessage)
{
    using (StreamWriter writer = File.AppendText("Log.txt"));
    {
        string loggerText = String.Format("{0} | Exception {1} happens with exception message: {2}", DateTime.Now.ToString(), exceptionText, exceptionMessage);
    }
}

In this case you can call it like this (for example in BackupClass.RestoreDatabase() method):

try {...}
catch(Exception e)
{
    LoggerClass.WriteLog("Restore DB error", e.Message);
    MessageBox.Show("DB restore is temporary unavailable. Please try again later.");
}